src/components/App/ScrollToTopOnMount.tsx

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from "react";
```

Comment:
This is awesome, way to find a solution to a not-so-obvious problem!!! And it's super clean!

Reply:
Seconded, good find!